projects/storefrontapp/src/app/spartacus/spartacus-b2c-configuration.module.ts

```diff
@@ -37,6 +37,7 @@ const baseSite = environment.epdVisualization
       context: {
         urlParameters: ['baseSite', 'language', 'currency'],
         baseSite: baseSite,
+        theme: ['santorini'],
```

Comment:
If the app needs a config to enable Santorini, then Santorini does not apply by default.  (as it seems it should)

Otherwise, if it does apply, we should remove this config.

Reply:
Removed